### PR TITLE
feature/145-register-page-design

### LINKED ISSUE #145: Register page design
Register page should be designed according to mockup below. 

Screenshot 2023-10-22 at 18 37 33

### PR DESCRIPTION
**Pull request by aykutkantas**
Resolves #145

### CODE REVIEW

File: backend/frontend/src/pages/landing/Register.js [hunk 1]
```diff
@@ -46,33 +52,124 @@ function Register() {
...
+                        boxSizing: 'border-box', // Include padding and borders in the total width and height
+                    }}
+                />
+            </div>
+              <div style={{width: 473, height: 59, left: 304, top: 320, position: 'absolute', background: 'white', borderRadius: 8, padding: 8}}>
+                    <TextField
+                            id="register-email"
+                            label="E-Mail"
+                            onChange={(e) => setEmail(e.target.value)}
+                            style={{
```
Comment (sadikkuzu) [inline backend/frontend/src/pages/landing/Register.js:113]: As far as i can see, register-username and register-email have same style codes. Is it possible to use css and className here in these input components?

File: backend/frontend/src/pages/landing/Register.js [hunk 2]
```diff
@@ -46,33 +52,124 @@ function Register() {
...
+                </div>
+            <div style={{width: 473, height: 59, left: 304, top: 400, position: 'absolute', background: 'white', borderRadius: 8, padding: 8}}>
+                {/* <label>Password:</label>
+                  <input type="password" className="form-control" onChange={(e) => setPassword(e.target.value)} /> */}
+                <TextField
+                                id="register-password"
+                                label="Password"
+                                type="password"
+                                onChange={(e) => setPassword(e.target.value)}
+                                style={{
```
Comment (sadikkuzu) [inline backend/frontend/src/pages/landing/Register.js:134]: same here

**Review by sadikkuzu — CHANGES_REQUESTED** [review]
Is it possible to use css and className for repetitive component styles?

**Comment by sadikkuzu** [comment]
it is better to follow our commit message conventions imho 🙏🏼 ☺️ 
https://github.com/SWE574-Fall2023-Group1/SWE574-Fall2023-G1/wiki/Conventions#commits

**Comment by sadikkuzu** [comment]
Nice design outcome, thanks.

actual

When i compare with figma design, i see header&footer font difference and button color difference.
expected
I just wanted to mention it.

---

In order to reproduce:

```shell
git pull
git checkout --track origin/feature/145-register-page-design 
make compose-up
```

http://localhost:3000/register

**Comment by sadikkuzu** [comment]
https://github.com/SWE574-Fall2023-Group1/SWE574-Fall2023-G1/blob/cd40dcecb37af5a6d169987d05bdbdc7e635a0ef/backend/frontend/src/pages/landing/Register.js#L8
Since there is an existing css file used in [Register.js](https://github.com/SWE574-Fall2023-Group1/SWE574-Fall2023-G1/blob/main/backend/frontend/src/pages/landing/Register.js#L8),
we may use [landing.css](https://github.com/SWE574-Fall2023-Group1/SWE574-Fall2023-G1/blob/main/backend/frontend/src/pages/landing/landing.css) for our style definitions.

**Review by dbaslan — APPROVED** [review]
LGTM, please update branch, make the requested changes and let's merge 👍

File: backend/frontend/src/pages/landing/Register.js (hunk 1 above)
Comment (aykutkantas) [reply backend/frontend/src/pages/landing/Register.js:113]: It's done!

**Review by sadikkuzu — APPROVED** [review]
Looks fantastic! ✨🚀

**Pull request merged**